USAGE.md

```diff
@@ -295,14 +296,38 @@ The `--template-settings` argument defines [index level settings](https://www.el
 
 For `template.json`, the `mappings` object is left empty: `{}`. Likewise the `properties` object remains empty in the `mapping.json` example. This will be filled in automatically by the script.
 
+#### OSS
+
+**IMPORTANT**: This feature is unnecessary for most users. Our default free distribution
```

Comment:
I've reworded the text here to make sure it's clear this will be unnecessary for most users. I don't think my original take made this clear enough.

One last 👀  @ebeahan and if you're good with this I'll merge & backport.